File: ass59/Program1.java
/*
1. Write Java program which accept String from user and display below pattern. 
Input :  Hello 
Output : H e l l o    H e l l o    H e l l o    H e l l o
*/
import java.lang.*;
import java.util.*;
class DemoString
{
	int WordFrequency(String str)
	{
		char arr[] = str.toCharArray();
		int size = arr.length;
        int i = 0, j= 0;
        
        for(i=0; i<size; i++)
        {
            for(j=0; j<size; j++)
            {
                System.out.print(""+arr[j]);
            }
            System.out.print("\n");
        }
        return 0;
    }
    
}
class Program1
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter string");

        String str = sobj.nextLine();
        sobj.close();
		DemoString dobj = new DemoString();

		dobj.WordFrequency(str);
	}
}

File: ass47/program1.c
/*
1. Write a recursive program which accept string from user and count white spaces. 
Input : HE llo WOr lD 
Output : 3 
*/
#include<stdio.h>
int Display(char *str)
{
    static int i = 0;
    if(*str!='\0')
    {   
        if(*str== '')
        {
            i++;
        }
        str++;
        Display(str);
    }
    return i;
}

int main()
{
    int iRet = 0;
    char arr[20];
    printf("Enter the string\n");
    scanf( "%s",arr);

    iRet = Display(arr);
    printf("%d",iRet);
    return 0;
}